server/cluster/unsafe_recovery_controller.go

```diff
@@ -485,6 +486,7 @@ func (u *unsafeRecoveryController) changeStage(stage unsafeRecoveryStage) {
 		output.Info = "Unsafe recovery enters exit force leader stage"
 		if u.err != nil {
```

Comment:
should set `u.err = nil` here, otherwise, it will call `handleErr` again after generate the exit force leader plan which change stage failed directly.

Reply:
Done

Reply:
Reverted, we don't need to set `u.err = nil` for the newest implementation, besides, the existing test case `TestFailed` actually expect the error is not cleaned up.

Reply:
Why? If it's not reset, it would hit `handleErr` on L356.

Reply:
Oh, that's right, seems we have another dilemma here, set `u.err = nil` makes `TestFailed` to fail, do you want to change the expectation?

Reply:
change whatever you want

Reply:
ping

Reply:
Done